python/delta_sharing/tests/test_rest_client.py

```diff
@@ -205,6 +205,14 @@ def test_query_table_metadata_partitioned_different_schemas(rest_client: DataSha
     )
 
 
+@pytest.mark.skipif(not ENABLE_INTEGRATION, reason=SKIP_MESSAGE)
+def test_query_table_version(rest_client: DataSharingRestClient):
+    response = rest_client.query_table_version(
+        Table(name="table1", share="share1", schema="default")
+    )
+    assert response.delta_table_version == 28
```

Comment:
This hardcoded value is decided by the current version of the table and if that table is updated this test will break, so maybe just assert some integer value is defined.

Reply:
added int assertion and that table version is positive